discovery-provider/src/solana/solana_client_manager.py

```diff
@@ -172,6 +185,8 @@ def _try_all(iterable, func, message, randomize=False):
     for index, value in items:
         try:
             return func(value, index)
+        except UnsupportedVersionError as e:
```

Comment:
A comment here explaining what this is about (and let's include a linear item for integration the new transaction type) would be great